packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,14 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement.
+class CapturedAccessibilityAnnouncement {
+  const CapturedAccessibilityAnnouncement(this.message, this.textDirection, {this.assertiveness = Assertiveness.polite});
+  final String message;
+  final TextDirection textDirection;
+  final Assertiveness? assertiveness;
```

Comment:
What's the use-case for having `null` assertiveness? The constructor defaults to polite, and so does the announcement API.

Reply:
Good point.